source/slurm/overview.rst

```diff
@@ -332,6 +332,169 @@ allowed.
 
 Run the command ``sbatch-limits -h`` to see the other available options.
 
+.. _memory-usage:
+
+Determine and specify a memory limit for your jobs
+==================================================
+
+You can use the "report-mem" command in your job to get memory usage
+information from your batch job. But this
+works only if you are able to successfully run your job to conclusion without
+failing.
+
+But if you don't know how much memory your application needs, you can "over
+estimate" or use an entire node to get a successful run and include the
+"report-mem" command in the job. To request all the available memory on the
+node for a serial job you can use the ``--mem=0`` option on the ``sbatch``
+command.
+
+If your jobs are failing with memory errors, it is possible that your
+application needs more memory than what you were giving for the job.
+
+In the case of serial jobs (which means you may have other jobs running on the
+same code and that your job is running), by default you get a certain amount of
+memory. If your application needs more memory than the default, you need to
+specify the memory needed by your job using the ``--mem=`` option. In general,
+for parallel jobs you do not need to specify a memory limit. You can specify
+the memory limit on the command line with using --mem option (for example
+``--mem=2g`` to specify 2 GB of memory) or as an #SBATCH directive within the
+job file.
+
+For parallel jobs it is not necessary to specify the
+memory requirement, but if each of your tasks requires more than its share of
+memory on the node, the only way to get more memory is to spread the same
+number of tasks on more nodes. The Memory High Water mark information
```

Comment:
"the only way to get more memory is to spread the same number of tasks on more nodes" -- how does one do this?

Reply:
There were sections that described how to do this in the old documentation, not sure if it made it into the new.
Here is a part of the image:

![image](https://github.com/user-attachments/assets/0ef8e43d-b14c-438f-b2d6-f6dcdf7657b1)

Reply:
added at #369 and subsequent